minecraft_servers/thorma90/manifest.json

```diff
@@ -2,20 +2,36 @@
     "server_name": "thorma90",
     "nice_name": "Thorma90 Netzwerk",
     "direct_ip": "thorma90.net",
+    "server_wildcards": [
+      "play.thorma90.net",
+      "mc.thorma90.net"
```

Comment:
Non-valid wildcards

Reply:
it works?

Reply:
its a ailases for minecraft

Reply:
Definition of a DNS wildcard: https://en.wikipedia.org/wiki/Wildcard_DNS_record
These are simple subdomains. This field is intended only for real wildcards that allow every subdomain.

Reply:
fixed